gtwrap/interface_parser/variable.py

```diff
@@ -0,0 +1,39 @@
+"""
+GTSAM Copyright 2010-2020, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+
+See LICENSE for the license information
+
+Parser classes and rules for parsing C++ variables.
+
+Author: Duy Nguyen Ta, Fan Jiang, Matthew Sklar, Varun Agrawal, and Frank Dellaert
+"""
+
+from .tokens import IDENT, SEMI_COLON
+from .type import TemplatedType, Type
+
+class Variable:
+    """
+    Rule to parse the variable members of a class.
```

Comment:
Update docstring? Also please update the example in the docstring to illustrate the various use cases. It makes understanding the parsing rule much easier.